### ISSUE TITLE
Timer initialization in Blockly

### ISSUE THREAD

**Issue by motib** [Mobsya/blockly#14]
When you use a timer in VPL it is initialized to 0 and this prevents spurious timer events. Blockly does not initialize the timer at all, so it is extremely difficult to use.

**Comment by stephanemagnenat** [comment]
I agree, blockly should have some initialization. Also it lacks an initialization area, an "on init" event.
